charts/hub-agent/crdbases/placement.kubernetes-fleet.io_clusterresourceoverridesnapshots.yaml

```diff
@@ -0,0 +1 @@
+../../../config/crd/bases/placement.kubernetes-fleet.io_clusterresourceoverridesnapshots.yaml
```

Comment:
Hi Zhiying! Considering that this is for the new agents only and v1alpha1 support is going to be away soon, this soft link is not necessary I believe. It can be directly added in the `crds` directory.